courses/urls.py

```diff
@@ -2,19 +2,18 @@
 from django.urls import include, re_path
 from rest_framework import routers
 
-from courses import views
+from courses.views import v1
 
 
 router = routers.SimpleRouter()
-router.register(r"programs", views.ProgramViewSet, basename="programs_api")
-router.register(r"courses", views.CourseViewSet, basename="courses_api")
-router.register(r"course_runs", views.CourseRunViewSet, basename="course_runs_api")
+router.register(r"programs", v1.ProgramViewSet, basename="programs_api")
+router.register(r"courses", v1.CourseViewSet, basename="courses_api")
+router.register(r"course_runs", v1.CourseRunViewSet, basename="course_runs_api")
 
 urlpatterns = [
+    re_path(r"^api/v1/", include(router.urls)),
     re_path(r"^api/", include(router.urls)),
     re_path(
-        r"^api/enrollments/",
-        views.UserEnrollmentsView.as_view(),
-        name="user-enrollments",
+        r"^api/enrollments/", v1.UserEnrollmentsView.as_view(), name="user-enrollments"
```

Comment:
I'm not sure what this enrollments API is for -- is it part of the anonymous public APIs? Why are we changing it?

Reply:
I'm not sure either, but I don't think this was changed. Only the reference to the `UserEnrollmentsView` was changed because the views were moved.